site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting the F5 Registry NGINX Plus Gateway Fabric Image
+description: "Learn how to pull an F5 NGINX Plus Gateway Fabric image from the official F5 Docker registry and upload it to your private registry. This guide covers the prerequisites, image tagging, and troubleshooting steps."
+weight: 100
+doctypes: ["install"]
+toc: true
+---
+
+{{<custom-styles>}}
+
+---
+
+## Before you begin
+
+Before you start, you'll need these installed on your machine:
+
+- [Docker v18.09 or higher](https://docs.docker.com/engine/release-notes/18.09/).
+- An NGINX Gateway Fabric subscription. Download both the certificate (*nginx-repo.crt*) and key (*nginx-repo.key*) from the [MyF5 Customer Portal](https://my.f5.com). Keep in mind that an NGINX Plus certificate and key won't work for for the steps in this guide.
+
+---
+
+## Set up Docker for F5 Container Registry
+
+Start by setting up Docker to commungfate with the F5 Container Registry located at `private-registry.nginx.com`. If you're using Linux, follow these steps to create a directory and add your certificate and key:
+
+```shell
+mkdir -p /etc/docker/certs.d/private-registry.nginx.com
+cp <path-to-your-nginx-repo.crt> /etc/docker/certs.d/private-registry.nginx.com/client.cert
+cp <path-to-your-nginx-repo.key> /etc/docker/certs.d/private-registry.nginx.com/client.key
+```
+
+The steps provided are for Linux. For Mac or Windows, consult the [Docker for Mac](https://docs.docker.com/desktop/faqs/general/#how-do-i-add-custom-ca-certificates) or [Docker for Windows](https://docs.docker.com/desktop/faqs/windowsfaqs/#how-do-i-add-custom-ca-certificates) documentation. For more details on Docker Engine security, you can refer to the [Docker Engine Security documentation](https://docs.docker.com/engine/security/).
+
+---
+
+## Pull the image
+
+Next, pull the image you need from `private-registry.nginx.com`. To find the correct image, consult the [Tech Specs guide](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/CHANGELOG.md).
```

Comment:
We should probably link to releases, not the CHANGELOG, right?

Reply:
I think the scope of this confusion is actually wider than it initially looks.

The repo README has a [technical specifications section}(https://github.com/nginxinc/nginx-gateway-fabric#technical-specifications), but pulling back even farther, there doesn't appear to be a good reference table for all available images.

For comparison, we currently direct people to look at the README on an otherwise sparse page on the documentation site:

https://docs.nginx.com/nginx-gateway-fabric/reference/technical-specifications/

The equivalent NGINX Ingress Controller page, however, lists the actual image names in tables based on registry.

https://docs.nginx.com/nginx-ingress-controller/technical-specifications/#supported-docker-images

The quick fix for this is to change the link to the README section above (I would also change the text to "consult the Technical Specifications" to match the heading", but tossing a ticket in the backlog to add a proper reference table for images would also great for the user experience.

Reply:
udpating

Reply:
updating